compiler+runtime/doc/install.md

```diff
@@ -66,6 +66,33 @@ yay -S jank-git   # if you installed jank-git
 yay -S jank-bin   # if you installed jank-bin
 ```
 
+## Nix
+### Install
```

Comment:
Each of these (Ubuntu, Homebrew, etc) has both an "Install" and an "Update" section. For Nix, we're missing that. So, how do folks who have used jank via nix before actually get the latest version?